mautic/mautic
# [FIX] Added default value to avoid null value exception

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" 
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Issue(s) addressed                     | Fixes #10231 

This should fix the failing API-Tests. A default value for the function was missing

#### Steps to test this PR:

To test with the API Library:

Assuming you are using DDEV and GitHub CLI:

`gh repo fork mautic/api-library`

select ‘yes’ to clone locally

`cd api-library`
`ddev start`

Type 'yes' to have DDEV set up a Mautic instance to test against (this will be in /mautic in the api-library directory)

Then run `ddev exec composer test` to run the tests.  Modify this command to run a specific test, eg. `ddev exec 
 composer test -- --filter testCreateGetAndDelete tests/Api/NotesTest.php`.

Modify this command to run all tests in one class, eg. `ddev exec composer test -- --filter test tests/Api/NotesTest.php`

This step may take some time ... go get a ☕ !

Once you can reproduce the issue, apply the PR:

`cd mautic`
`gh pr checkout 10259`
`cd ..`
`ddev exec composer test`

Tests should now pass.
